src/Spice86/Spice86DependencyInjection.cs

```diff
@@ -499,13 +498,10 @@ internal Spice86DependencyInjection(Configuration configuration, MainWindow? mai
 
             VideoCardViewModel videoCardViewModel = new(vgaRenderer, videoState);
 
-            CpuViewModel cpuViewModel = new(state, memory, pauseHandler, uiDispatcher);
-
+            ICpuViewModel cpuViewModel = cpuViewModelSupplier.Invoke(uiDispatcher);
```

Comment:
Using an Action makes the code less understandable (at least for me).

Can the code be more direct ? Like use a local function, or an extension method ?